docker-compose.yaml

```diff
@@ -1,11 +1,11 @@
-version: '3'
+version: '3.8'
 services:
   portal:
     build: .
     env_file: .env
 
   nginx:
-    image: nginx:alpine
+    image: nginx:1.20-alpine
```

Comment:
Again, we should specify the full image version if we want reproducibility, e.g. nginx 1.20.2

Reply:
Same as [here](https://github.com/accrescent/devportal/pull/2#discussion_r788247997), I don't think it's necessary, and it can even be inconvenient. But I can change it.